src/pages/about.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: '../layouts/Legal.astro'
+title: 'Übersicht'
+---
+
+# Was Metropoliplan (MEP) ist
+
+## Zweck von Metropoliplan
+
+Metropoliplan wurde für die speziellen Bedürfnisse des öffentlichen Zeugnisgebens in Metropolregionen entwickelt. Zudem wurden auch Prozesse für das Zeugnisgeben auf Rastplätzen integriert.  
+
+Diese Software ist nicht für das Trolley Zeugnisgeben auf Versammlungsebene gedacht.
+
+Die Zielgruppe von Metropoliplan sind speziell Organisationsteams, die große Projekte mit vielen Versammlungen koordinieren müssen. In diesen Größenordnungen funktioniert die Selbstorganisation nicht mehr und erfordert mehr Einflußnahme.
+  
+## User Interface
+
+/UserAndAdminDashboard.png
+
+User Dashboard das sowohl Neuigkeiten anzeigt, als auch aktuelle Bewerbungen und Zuteilungen anzeigt. 
+
+/PlannerCalender.png
+
+Admin Ansicht auf den Planungskalender. Sowohl die Bewerbung als auch Planung funktioniert intuitiv wie bei einem Microsoft Outlook Kalender.
+
+## Funktionen
+
+### Metropoliplan ist:
+
+* eine zentral gehostete Website (d.h. Ihr müsst keinen eigenen Betrieb gewährleisten)
+* mit jedem modernen Browser nutzbar (keine Apps nötig)
+* eine mehrsprachige Lösung:
+  * Englisch
+  * Deutsch
+  * Französisch
+  * Spanisch
+  * Russisch
+  * (weitere Sprachen auf Anfrage)
+* Mehrmandantenfähig (d.h. wenn Ihr Verkündiger habt die in mehreren Projekten tätig sind, können sie die gleichen Login Daten verwenden)
+* kostenfrei (aber durch eure freiwilligen Spenden unterstützt)
+* optimiert für große und sehr große Projekte (50-5000 User)
+* Datenschutzkonform (DSGVO)
```

Comment:
Statt langer Liste vielleicht nur Hinweis, dass unser Datenschutz- und IT-Sicherheitskonzept bei Bedarf eingesehen werden kann. (Nach Unterzeichnung eines NDA mit Nasenblut 😉)